sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -275,7 +275,7 @@ def write_logged_features(
     def offline_write_batch(
         config: RepoConfig,
         table: FeatureView,
```

Comment:
nit this shouldn't be called table it should be `feature_view` or something.

Reply:
i was adhering to how we defined in online write batch but ill change it.